models/DataObject/Localizedfield/Dao.php

```diff
@@ -163,11 +163,9 @@ public function save(array $params = []): void
                 'language' => $language,
             ];
 
-            if ($container instanceof DataObject\Objectbrick\Definition) {
+            if ($container instanceof DataObject\Objectbrick\Definition || $container instanceof DataObject\Fieldcollection\Definition) {
                 $insertData['fieldname'] = $context['fieldname'];
-            } elseif ($container instanceof DataObject\Fieldcollection\Definition) {
-                $insertData['fieldname'] = $context['fieldname'];
-                $insertData['index'] = $context['index'];
+                $insertData['index'] = $context['index'] ?? 0;
```

Comment:
but according brick tables do not have the index column, right? 
so, there is actually no sense in setting that value?

Reply:
The column is there, but not used, but still part of the primary key. 
It´s the same code for object bricks and field collections:
https://github.com/pimcore/pimcore/blob/f3275ffd2352e437a67df8bf7edaa2f675cd3831/models/DataObject/Localizedfield/Dao.php#L801

Reply:
true...